examples/data-objects/table-document/src/table.ts

```diff
@@ -3,18 +3,15 @@
  * Licensed under the MIT License.
  */
 
-import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { Serializable } from "@fluidframework/datastore-definitions";
 import { ICombiningOp, PropertySet } from "@fluidframework/merge-tree";
-import {
-    Jsonable,
-    JsonablePrimitive,
-} from "@fluidframework/datastore-definitions";
 
-export type TableDocumentItem = Jsonable<JsonablePrimitive | IFluidHandle>;
+export type TableDocumentItem = Serializable;
```

Comment:
I'm not sure what this even does. is this any different than just using any?

Reply:
In this draft, I was lazy with the conversion of this deprecated class.

The right change would be `type TableDocumentItem<T> = Serializable<T>`, with the generic argument for <T> plumbed through everywhere...  but I'm not sure it's worth the noise at this point.

Reply:
From your other comment below, I see how this adds more confusion than clarity.  'any' it is.  thx!